gateway-js/src/__tests__/queryPlanCucumber.test.ts

```diff
@@ -25,11 +25,8 @@ features.forEach((feature) => {
         let queryPlan: QueryPlan;
         let options: BuildQueryPlanOptions = { autoFragmentization: false };
 
-        const { schema, errors, queryPlannerPointer } = getFederatedTestingSchema();
-
-        if (errors && errors.length > 0) {
-          throw new GraphQLSchemaValidationError(errors);
-        }
+        // throws on composition errors
```

Comment:
This `if` statement is redundant, as `getFederatedTestingSchema` already handled this same logic internally.